pkg/kapp/resourcesmisc/apps_v1_deployment_test.go

```diff
@@ -65,3 +65,64 @@ func buildDep(resourcesBs string, t *testing.T) *ctlresm.AppsV1Deployment {
 
 	return ctlresm.NewAppsV1Deployment(newResources[0], newResources[1:])
 }
+func TestIsDoneApplying(t *testing.T) {
```

Comment:
Let's move the test above the `buildDep` helper function. Also, let's rename it to `TestAppsV1DeploymentReplicaFailure`.